src/euphorie/content/browser/survey.py

```diff
@@ -520,3 +522,51 @@ def tools(self):
                 # we're done if we found at least one measure
                 break
         return tools
+
+
+class ListLinks(BrowserView):
+    attributes_checked = [
+        "description",
+        "introduction",
+        "solution_direction",
+        "legal_reference",
+        "action",
+        "action_plan",
+        "prevention_plan",
+        "requirements",
+    ]
+    url_regex = re.compile(
+        r"https?://[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b"
+        r"(?:[-a-zA-Z0-9()@:%_+.~#?&/=]*)"
+    )
```

Comment:
This is probably fine.  Alternative idea: look for contents of href/src, so `="http..."`, with at the dots anything except a double quote or a space.  Something like this:

```
>>> re.compile(r'="(https?://[^ "]*)"').match('="https://some.url"').groups()
('https://some.url',)
```